### PR TITLE
Update exceptions.py

### PR DESCRIPTION
**Pull request by jakubczakon**
updated links after docs change

### CODE REVIEW

File: neptune/new/exceptions.py [hunk 1]
```diff
@@ -492,7 +492,7 @@ def __init__(self):
 
 class NeptuneConnectionLostException(NeptuneException):
     def __init__(self):
-        message = """
+        message = """https://docs.neptune.ai/administration/workspace-project-and-user-management/projects
```
Comment (Raalsky) [inline neptune/new/exceptions.py:495]: Something wrong went here I think. There should be anything before the exception header.

**Review by Herudaio — COMMENTED** [review]
Apart from what @Raalsky already pointed out looks good to me.

**Review by pkasprzyk — DISMISSED** [review]
See @Raalsky's comment at https://github.com/neptune-ai/neptune-client/pull/651#discussion_r688251968

**Comment by jakubczakon** [comment]
Fixed @Herudaio @pkasprzyk

**Review by Herudaio — APPROVED** [review]

**Pull request merged**